src/mcp.ts

```
@@ -202,3 +205,318 @@ export function createMcpHttpApp(options: LoadOptions = defaultOptions): Hono {
+
+				await Promise.all([
+					client.connect(clientTransport),
+					server.connect(serverTransport),
+				]);
+
+				const result = await client.callTool({
+					name: 'daily',
+					arguments: { mode: 'auto' },
+				});
+
+				expect(result).toHaveProperty('content');
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)